oauth2client/contrib/__init__.py

```diff
@@ -4,3 +4,4 @@
 library but provide additional functionality. These modules are intended to
 make it easier to use oauth2client.
 """
+
```

Comment:
Needless newline insertion? Why is this file part of this commit? Did IDEA/PyCharm/another tool do this?